### ISSUE TITLE
🐛 Biome encountered an unexpected error

### ISSUE THREAD

**Issue by HermanBergsholm** [biomejs/biome#6702]
### Environment information

```block
CLI:
  Version:                      2.0.0
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v23.11.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         npm/10.9.2

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  false

Workspace:
  Open Documents:               0
```

### What happened?

2025-07-04 13:49:35.864 [info] [Error - 1:49:35 PM] Server process exited with code 0.
2025-07-04 13:49:35.864 [info] [Info  - 1:49:35 PM] Connection to server got closed. Server will restart.
2025-07-04 13:49:35.864 [info] true
2025-07-04 13:49:35.888 [info] Biome encountered an unexpected error

This is a bug in Biome, not an error in your code, and we would appreciate it if you could report it to https://github.com/biomejs/biome/issues/ along with the following information to help us fixing the issue:

Source Location: crates/biome_service/src/settings.rs:78:14
Thread Name: tokio-runtime-worker
Message: You must have at least one workspace.

2025-07-04 13:49:35.889 [info] [Error - 1:49:35 PM] Server process exited with code 0.
2025-07-04 13:49:35.889 [info] [Error - 1:49:35 PM] The biome server crashed 5 times in the last 3 minutes. The server will not be restarted. See the output for more information.

### Expected result

It should not unexpectedly shut down (?)

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
